core/src/components/login/LoginForm.vue

```diff
@@ -203,10 +203,10 @@ export default {
 
 	mounted() {
 		if (this.username === '') {
-			this.$refs.user.focus()
+			this.$refs.user.$refs.inputField.$refs.input.focus()
```

Comment:
This targets an `NcInputField` which has no inputField ref.
```suggestion
			this.$refs.user.$refs.input.focus()
```

Reply:
Sorry, nevermind, it targets an `NcTextField` which of course has an inputField ref. So all good.

Reply:
Thanks for your review :+1: